recipes/spix/all/conanfile.py

```diff
@@ -0,0 +1,131 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.files import apply_conandata_patches, export_conandata_patches, get, copy, rm, rmdir, replace_in_file
+from conan.tools.build import check_min_cppstd
+from conan.tools.scm import Version
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+import os
+
+
+required_conan_version = ">=1.52.0"
+
+
+class SpixConan(ConanFile):
+    name = "spix"
+    description = "UI test automation library for QtQuick/QML Apps"
+    license = "MIT"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/faaxm/spix"
+    topics = ("automation", "qt", "qml", "qt-quick", "qt5", "qtquick", "automated-testing", "qt-qml", "qml-applications")
+    settings = "os", "arch", "compiler", "build_type"
+    options = {
+        "shared": [True, False],
+        "fPIC": [True, False],
+        "qt_major": [5, 6],
```

Comment:
Poke our Qt community leader @ericLemanissier, this option does not exist in CCI so I am wondering what the normal way of doing it would be?

best I found it https://github.com/conan-io/conan-center-index/blob/06927ee56288a37d5cb44ca6a76fcd63d09df9af/recipes/poppler/all/conanfile.py#L315

Reply:
@fdgStilla please, use Prince's suggestion if you don't have a real limitation. We don't try to add complexity if it can be solved "automatically" by a dependency